src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,28 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post }) => (
+  <>
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{post.title}</h3>
+
+        <p>
+          {' Posted by  '}
+
+          <UserInfo user={post.user} />
+        </p>
+      </div>
+
+      <p className="PostInfo__body">{post.body}</p>
+
+      <hr />
+
+      {post.comments.length === 0 ? (
+        <b data-cy="NoCommentsMessage">No comments yet</b>
+      ) : (
+        <CommentList comments={post.comments} />
+      )}
```

Comment:
Better put wright comment first
```suggestion
      or  {!!post.comments.length ? (
      {post.comments.length > 0 ? (
        <CommentList comments={post.comments} />
      ) : (
        <b data-cy="NoCommentsMessage">No comments yet</b>
      )}
```